include/boost/unordered/unordered_flat_map.hpp

```diff
@@ -53,6 +53,11 @@ namespace boost {
           return kv.first;
         }
 
+        static moved_type move(init_type& x)
+        {
+          return {std::get<0>(std::move(x)), std::get<1>(std::move(x))};
+        }
```

Comment:
Why not `{std::move(x.first), std::move(x.second)};`?

Same for other places where this construct appears.

Reply:
That's not the same, see https://gcc.gnu.org/bugzilla/show_bug.cgi?id=108952

Reply:
Ah wait, it's the same here. :-)